docker/mbari_wec/Dockerfile

```
@@ -119,12 +119,14 @@ RUN /bin/bash -c 'source /opt/ros/humble/setup.bash \
 ENV SETUP_SH /home/$USERNAME/setup.bash
 RUN touch ${SETUP_SH} \
  && chmod 755 ${SETUP_SH} \
- && echo ". ${MBARI_WEC_WS}/install/setup.bash" >> ${SETUP_SH}
+ && echo ". ${MBARI_WEC_WS}/install/setup.bash" >> ${SETUP_SH} \
+ && echo "export SDF_PATH=\$GZ_SIM_RESOURCE_PATH" >> ${SETUP_SH}
 ENV RUN_SH /home/$USERNAME/run_simulation.bash
 RUN touch ${RUN_SH} \
  && chmod 755 ${RUN_SH} \
  && echo ". ${MBARI_WEC_WS}/install/setup.bash" >> ${RUN_SH} \
+ && echo "export SDF_PATH=\$GZ_SIM_RESOURCE_PATH" >> ${RUN_SH} \
```

Hmm as the run file is getting bigger, I wonder if we can just call the setup file from it, like 
```suggestion
 && echo ". ${SETUP_SH}" >> ${RUN_SH} \
```
Saves some duplication.
I haven't tested this though.